en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md

```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
     }
     ```
 
+??? note "campaignCriteriaMutate"
+    Creates, updates, or removes criteria. Operation statuses are returned.
```

Comment:
Shall we follow the same format as https://github.com/wso2/docs-mi/pull/1003/files#diff-c21ad6e833416d1a7b70bc9961744e37e6da384d18f2016df0a0d9bf43ac0859L655